### DISCUSSION TITLE
The menu and extension menus has dark text on dark background in both dark and light modes, how to fix? or is this a bug?

### DISCUSSION THREAD

**Discussion by ramizmortada** [zen-browser/desktop#2197] [Q&A]
![image](https://github.com/user-attachments/assets/bf431649-2765-468a-be05-e2cc1e37c713)
![image](https://github.com/user-attachments/assets/714c1249-e234-4ff2-9a04-99d30030032a)

**Comment by ramizmortada** [comment]
It was a theme issue fix when I disabled the theme

**Comment by vuongtrunghieu** [comment]
Hey, to anyone who's looking for the issue and fix, I found this on the [discord server](https://discord.com/channels/1088172780480114748/1307809809856204800/1308084352571146271)

`about:config` → `browser.tabs.allow_transparent_browser` to `false`, then reopen your website in a new tab.
